ui/src/cpu_inspector.rs

```diff
@@ -67,7 +71,8 @@ impl View for CpuInspector {
 
                 self.thread_handle = Some(thread::spawn(move || {
                     while play_clone.load(std::sync::atomic::Ordering::Relaxed) {
-                        gba_clone.lock().unwrap().cpu.step();
+                        thread::sleep(Duration::from_millis(1000 / FPS));
+                        acquire_lock!(gba_clone, gba => {gba.step()});
```

Comment:
I'm not sure about the semantics. Does 1 fps corresponds to a cpu cycle + lcd refresh?

Reply:
It is not correct. It put it just to avoid  GUI slowdown. We need to manage this thread, because I think that it is better put in pause the execution when an user is in the GUI.  Maybe I'll address it later. For now it is an escamotage.